.changeset/wet-bars-report.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/backend-plugin-api': minor
+'@backstage/backend-defaults': patch
```

Comment:
Thinking it could be nice with a changeset for each of these, the current only mostly applies to `plugin-api`